docs/src/bpmn-workflows/message-events.md

```diff
@@ -55,6 +55,13 @@ XML representation:
 </bpmn:intermediateCatchEvent>
 ```
 
+### Boundary Events
+
+When attached to the boundary of an activity, an intermediate message catch event behaves in two ways:
+
+If it is non-interrupting, it will spawn a new token which will take all outgoing sequence flows, applying any defined output mapping with the message payload.
+If it is interrupting, however, it will first terminate the activity before spawning the token.
```

Comment:
Hm I'm not a native speaker but for me it sounds a bit weird, but I' dont know. Can we remove the however?

Reply:
@npepinpe here, I think it works better without "however". 

Maybe at the end of the first sentence, you could add a second sentence with more explanation. E.g.:

If it is non-interrupting, it will spawn a new token which will take all outgoing sequence flows, applying any defined output mapping with the message payload. It will _not_ terminate the activity that it's attached to.

Otherwise, I'd just drop the "however".